lib/link_checker.rb

```diff
@@ -49,7 +49,7 @@ def generate_report
 
   def link_checker_api
     @link_checker_api ||= GdsApi::LinkCheckerApi.new(
-      Plek.current.find("link-checker-api"),
+      Plek.new.find("link-checker-api"),
```

Comment:
I think you can just use `Plek.find` here. See: https://github.com/alphagov/plek/blob/0f9c8e8bccc53f01d180a2fe80aabfc853dcbdc6/lib/plek.rb#L73

Reply:
That is cleaner!